core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerFacade.java

```diff
@@ -97,9 +97,13 @@ private MediaPlayer createMediaPlayer(SurfaceHolder surfaceHolder, Uri videoUri)
         return mediaPlayer;
     }
 
+    protected MediaPlayer createMediaPlayer() {
```

Comment:
🤢  but this is a temporary step so that we can test the behaviour as is, later we will refactor this class.